app/dashboard/models.py

```diff
@@ -6111,6 +6111,8 @@ class PassportStamp(SuperModel):
     user = models.ForeignKey(User, related_name='passport_stamps', on_delete=models.CASCADE, null=True, db_index=True)
     passport = models.ForeignKey(Passport, related_name='stamps', on_delete=models.CASCADE, null=True)
     stamp_id = models.CharField(unique=True, null=False, blank=False, max_length=100)
+    stamp_provider = models.CharField(null=False, blank=False, default=False, max_length=256)
```

Comment:
`default=False` was this on purpose? I think `default` should be a proper string value.
Or maybe make it null-able and set it to None initially?

Reply:
Also a minor thing: could we add this field to the stamp list in the admin?